docs/topics/request-response.rst

```diff
@@ -54,7 +54,7 @@ Request objects
       ``body`` is not given, an empty string is stored. Regardless of the
       type of this argument, the final value stored will be a ``str`` (never
       ``unicode`` or ``None``).
-    :type body: str or unicode
+    :type body: str
```

Comment:
technically, this is no longer correct in Python 2, but let's not worry about it